docs/development-changelog/balance-snippet.md

```
@@ -6,23 +6,57 @@ permalink: development/changelog/balance-snippet
-_Information about how to structure a balance snippet_
+The game just renders the text as is, so we need to keep that in mind.
+Unit or explanation first?
+The unitID is also used to infer if the snippet is supposed to go into the land, navy, air, structures or enhancements category.
+if the change is an enhancement, the unitID is enhancements/faction/name
+The md to lua converter should remove the {: .unit-change} and the (unitID)
+
+
 
 The snippet file should be named `balance.<PR Number>.md`.
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)